doc/rdo-packaging.txt

```diff
@@ -241,7 +242,7 @@ Run `delorean` for the package you are trying to build.
 $> delorean --config-file projects.ini --local --package-name openstack-horizon
```

Comment:
Unrelated to this change, this caught my eye: SRPM name is not openstack-horizon, I guess that's order issue in rdo.yml:
- project: horizon
  package: python-django-horizon
  conf: core
^^^ "package" field should come here, after "conf" template is applied
  distgit: ssh://pkgs.fedoraproject.org/python-django-horizon.git
  maintainers:
  - mrunge@redhat.com
  conf: core
In any case, Horizon is still not working in Delorean, we need to fix that too :(